<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proc_MainPage</class>
 <widget class="QWidget" name="Proc_MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,100">
   <property name="spacing">
    <number>13</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_topBar_container" native="true">
     <layout class="QHBoxLayout" name="layout_topBar_main">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_topBar_totalNum" native="true">
        <layout class="QVBoxLayout" name="layout_topBar_1">
         <item>
          <widget class="QLabel" name="label_topBar_totalNum_title">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>公司员工数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_topBar_totalNum_num">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_topBar_checkedNum" native="true">
        <layout class="QVBoxLayout" name="layout_topBar_2">
         <item>
          <widget class="QLabel" name="label_topBar_checkedNum_title">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>今日签到人数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_topBar_checkedNum_num">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_topBar_requestNum" native="true">
        <layout class="QVBoxLayout" name="layout_topBar_3">
         <item>
          <widget class="QLabel" name="label_topBar_requestNum_title">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>待处理请求</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_topBar_requestNum_num">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_topBar_personChgNum" native="true">
        <layout class="QVBoxLayout" name="layout_topBar_4">
         <item>
          <widget class="QLabel" name="label_topBar_personChgNum_title">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>人员变动数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_topBar_personChgNum_num">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_bottom" stretch="5,2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="layout_chartMain">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_pieChart_container" native="true">
         <layout class="QHBoxLayout" name="layout_widgetPieChartContainer">
          <item>
           <widget class="QWidget" name="widget_showPieChart_aRate" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_showPieChart_bRate" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_barChart_container" native="true">
         <layout class="QHBoxLayout" name="layout_widgetBarChartContainer" stretch="3,1">
          <item>
           <widget class="QWidget" name="widget_showBarChart" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_showManagerInfo" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_showManagerName">
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>管理者名字:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LEdit_showManagerName"/>
             </item>
             <item>
              <widget class="QLabel" name="label_showManagerId">
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>管理者ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LEdit_showManegerId"/>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget_showDate_container" native="true">
       <layout class="QVBoxLayout" name="layout_showDateMain" stretch="3,2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_showCurDate" native="true">
          <layout class="QVBoxLayout" name="layout_widget_showCurDate" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_showDate_month">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>MonthLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_showDate_day" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
